### PR TITLE
NOISSUE: springcleaning

### PR DESCRIPTION
**Pull request by yellowmegaman**
* all pipelines are now using manifests imported from gitops repo
* kube-deploy-tool is removed
* existing old and outdated manifests are removed

### CODE REVIEW

**Review by sexyhamster — DISMISSED** [review]

File: .github/workflows/consensus-checks.yaml [hunk 1]
```diff
@@ -3,110 +3,106 @@ on:
...
-        run: bin/kube-deploy-tool --config=scripts/deploy/kube-deploy-tool/config.yaml
-        working-directory: ${{ env.CORE_PATH }}
-      - name: Aggregate logs into a single archive
+          TARGET: ${{matrix.target}}
+        run: |
+          echo "::set-env name=INSOLAR_TAG::${GITHUB_SHA:0:8}"
+          echo "::set-env name=INSOLAR_NETWORK_SIZE::${TARGET}"
+          echo "::set-env name=INSOLAR_NAMESPACE::${GITHUB_SHA:0:8}-${TARGET}"
+          echo "::set-env name=INSOLAR_DESIRED_UPTIME::300"
+          echo "::set-env name=INSOLAR_BRANCH::$(echo "pull-$(echo ${{github.ref}} | cut -d '/' -f3)")"
```
Comment (ita-sammann) [inline .github/workflows/consensus-checks.yaml:60]: Why there is always `pull-` prefix in the env variable?

File: .github/workflows/master-checks.yaml [hunk 2]
```diff
@@ -147,48 +147,54 @@ jobs:
...
+        run: git config --global url."https://${{secrets.INS_SERVICE_PAT}}@github.com/".insteadOf "https://github.com/"
+      - name: Preparation - set prerequisite variables, full sha is ${{github.sha}}
+        env:
+          TARGET: ${{matrix.target}}
+        run: |
+          echo "::set-env name=INSOLAR_TAG::${GITHUB_SHA:0:8}"
+          echo "::set-env name=INSOLAR_NETWORK_SIZE::${TARGET}"
+          echo "::set-env name=INSOLAR_NAMESPACE::${GITHUB_SHA:0:8}-${TARGET}"
+          echo "::set-env name=INSOLAR_DESIRED_UPTIME::300"
+          echo "::set-env name=INSOLAR_BRANCH::$(echo "pull-$(echo ${{github.ref}} | cut -d '/' -f3)")"
```
Comment (ita-sammann) [inline .github/workflows/master-checks.yaml]: Same as above, always `pull-` prefix.

File: .github/workflows/linux-checks.yaml [hunk 3]
```diff
@@ -166,36 +169,39 @@ jobs:
...
+        run: git config --global url."https://${{secrets.INS_SERVICE_PAT}}@github.com/".insteadOf "https://github.com/"
+      - name: Preparation - set prerequisite variables, full sha is ${{github.sha}}
+        env:
+          TARGET: ${{matrix.target}}
+        run: |
+          echo "::set-env name=INSOLAR_TAG::${GITHUB_SHA:0:8}"
+          echo "::set-env name=INSOLAR_NETWORK_SIZE::${TARGET}"
+          echo "::set-env name=INSOLAR_NAMESPACE::${GITHUB_SHA:0:8}-${TARGET}"
+          echo "::set-env name=INSOLAR_DESIRED_UPTIME::300"
+          echo "::set-env name=INSOLAR_BRANCH::$(echo "pull-$(echo ${{github.ref}} | cut -d '/' -f3)")"
```
Comment (ita-sammann) [inline .github/workflows/linux-checks.yaml:182]: Same as above, always `pull-` prefix.

File: .github/workflows/linux-checks.yaml [hunk 4]
```diff
@@ -166,36 +169,39 @@ jobs:
...
+          echo "::set-env name=INSOLAR_NETWORK_SIZE_INT::$(echo $TARGET | awk '{gsub(/[^[0-9]]*/,"");print}')"
```
Comment (ita-sammann) [inline .github/workflows/linux-checks.yaml:183]: Do we need network size here?

File: .github/workflows/linux-checks.yaml (hunk 3 above)
Comment (ita-sammann) [reply .github/workflows/linux-checks.yaml:182]: Do we need this var at all here?

File: .github/workflows/master-checks.yaml (hunk 2 above)
Comment (ita-sammann) [reply .github/workflows/master-checks.yaml]: Do we need this var at all here?

**Review by ita-sammann — CHANGES_REQUESTED** [review]

File: .github/workflows/linux-checks.yaml (hunk 4 above)
Comment (yellowmegaman) [reply .github/workflows/linux-checks.yaml:182]: Not exactly a size here, but can be used like one, later on

**Review by ita-sammann — APPROVED** [review]

**Review by twwlf — APPROVED** [review]

**Review by ivanshibitov — APPROVED** [review]

**Pull request merged**